ui/src/app/base/components/ActionForm/ActionForm.tsx

```diff
@@ -96,31 +98,40 @@ const getLabel = (
 };
 
 export type Props<V, E = null> = Omit<
-  FormikFormProps<V, E>,
+  FormikFormContentProps<V, E>,
   "buttonsAlign" | "saved" | "saving" | "savingLabel" | "submitLabel"
 > & {
   actionName: string;
   loaded?: boolean;
+  initialTouched?: FormikConfig<V>["initialTouched"];
+  initialValues: FormikConfig<V>["initialValues"];
   modelName: string;
+  onSubmit: FormikConfig<V>["onSubmit"];
```

Comment:
`ActionForm` intercepts `onSubmit` so we can't wrap these forms in `Formik` and because we're already spreading on `FormikFormContent` we need to explicitly capture the `Formk` props.